src/monitor/Makefile

```diff
@@ -20,7 +20,14 @@ PG_CONFIG ?= pg_config
 PGXS = $(shell $(PG_CONFIG) --pgxs)
 USE_PGXS = 1
 
+.PHONY: clenaup-before-install
```

Comment:
Typo in cleanup ; also, what about naming it `maintainer-clean` as in Postgres upstream?